fix for "malloc.c:389: error: return type is an incomplete type"
inspired by http://gnats.netbsd.org/43870
author: Kenneth Hoste (HPC-UGent)
--- graphviz-2.40.1/lib/vmalloc/features/vmalloc.orig	2018-04-19 15:15:03.568712102 +0200
+++ graphviz-2.40.1/lib/vmalloc/features/vmalloc	2018-04-19 15:15:13.618836531 +0200
@@ -11,9 +11,6 @@
 sys stat
 typ ssize_t
 hdr malloc
-lib mallopt
-lib mallinfo
-lib mstats
 hdr dlfcn
 
 std	malloc note{ stuck with standard malloc }end noexecute{
